Rakefile

```diff
@@ -21,4 +24,9 @@ RSpec::Core::RakeTask.new(:spec_standalone) do |t|
   t.pattern = 'spec/scripts/**/*_spec.rb'
 end
 
+# Package Tasks
+Simp::Rake::Pkg.new(File.dirname(__FILE__))
+
+# Acceptance Tests
```

Comment:
We don't have acceptance tests, so this is not needed, currently.  Was this added in in case we want to add acceptance tests?